etc/kayobe/ansible/octavia-amphora-image-register.yml

```diff
@@ -51,7 +51,7 @@
     - name: Ensure Octavia Amphora image is renamed
       vars:
         ansible_python_interpreter: "{{ venv }}/bin/python"
-      shell:
+      ansible.builtin.command:
```

Comment:
Should this be `ansible.builtin.comand`? You'd replaced `shell` with `ansible.builtin.shell` elsewhere.

Reply:
I think that's this rule: https://ansible.readthedocs.io/projects/lint/rules/command-instead-of-shell/#command-instead-of-shell